include/alpaka/kernel/KernelFunctionAttributes.hpp

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2022 Benjamin Worpitz, Bert Wesarg, René Widera, Sergei Bastrakov, Bernhard Manfred Gruber
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+#pragma once
+
+#include <cstddef>
+
+namespace alpaka
+{
+    //! Kernel function attributes struct. Attributes are filled by calling the API of the accelerator using the kernel
```

Comment:
please add a comment with the behaviour if an entry is not filled by the underlaying api e.g. on CPU

Reply:
please add a description to which values like localSizeBytes will be set if those values are not available for a backend/accelerator.

Reply:
For the CPU backends (serial, threads, TBB, OMP2), `maxDynamicSharedSizeBytes` should be set to the value of `alpaka::BlockSharedDynMemberAllocKiB * 1024`, as defined in `alpaka/block/shared/dyn/BlockSharedDynMemberAllocKiB.hpp`.

Reply:
Done, thanks, good info.